website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/provider_observer.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: ProviderObserver
+---
+
+[ProviderObserver] écoute les changements d'un ProviderContainer.
+
+Pour l'utiliser, étendez (extend) la classe ProviderObserver et surchargez (override) 
+la méthode que vous souhaitez utiliser.
+
+- `didAddProvider` est appelé chaque fois qu'un provider a été initialisé, et la valeur exposée est value.
+- `didDisposeProvider` est appelé chaque fois que le provider A a été éliminé.
+- `didUpdateProvider` est appelé à chaque fois que mes providers émettent une notification.
+- `mayHaveChanged` est appelé lorsque la dépendance d'un provider a changé, mais il n'est pas encore certain que la valeur calculée change.
+
+:::info
+Il est possible que mayHaveChanged soit appelé, sans que didUpdateProvider soit appelé, 
+par exemple lorsqu'un Provider est recalculé mais renvoie une valeur == à la précédente.
+:::
```

Comment:
```suggestion
```

Reply:
This life-cycle was removed. The en doc is out of date

Reply:
Sorry, I didn't pay attention to changes, I was focused more on the translation.

I will update (remove 'hayHaveChanged' part) in the English documentation as well.

Other than that is the french translation for the **Concepts** section all good? So I can continue doing the rest of the doc.